insights/parsers/httpd_conf.py

```diff
@@ -109,6 +110,7 @@ class HttpdConf(LegacyItemAccess, Parser):
     """
 
     def __init__(self, *args, **kwargs):
+        deprecated(HttpdConf, "Import HttpdConfTree from insights.combiner.httpd_conf instead.")
```

Comment:
Same comment as above, there is no way to know this is deprecated until you look at the code.